rudder-web/src/main/scala/com/normation/rudder/web/components/TechniqueEditForm.scala

```diff
@@ -210,7 +212,9 @@ class TechniqueEditForm(
         case e:EmptyBox => NodeSeq.Empty
```

Comment:
You are right, it use empty as "there is no active technique", so don't log on emptybox but only on failure